guides/common/modules/con_how-and-why-project-collects-usage-metrics.adoc

```diff
@@ -0,0 +1,14 @@
+[id="how-and-why-{project-context}-collects-usage-metrics"]
+= How and why {Project} collects usage metrics
```

Comment:
What about changing this to something like **Overview of usage metrics collection in {Project}**?

Reply:
Renamed.